doc/whats-new.md

```diff
@@ -1,358 +1,387 @@
-extended to accept a `min_mag` parameter to filter out the small
-magnitudes and reduce substantially the download time.
-
-We deprecated the XML exporter for the ruptures, since it is extremely
-verbose and inefficient. In the future we will replace it with a better
-solution. For the moment you can still use it or, better, you can use
-the `/extract/rupture_info` binary API.
-
-In `ebrisk` calculations we have clearly split the outputs aggregated
-by tag from the total outputs: the new names are 'Aggregate
-Losses/Aggregate Loss Curves', and 'Total Losses/Total Loss Curves'
-respectively. This avoids a lot of confusion, because in the past the
-total loss curves where are also called aggregate.  The total outputs
-are computed from the event loss table while the aggregate outputs are
-computed from the asset loss table. We have also removed the outputs
-`agg_maps-rlzs` and `agg_maps-stats` that were only accidentally
-exported in engine 3.7.
+The disaggregation calculator can compute the mean disaggregation,
+if multiple realizations are specified by the user in the job.ini. This
+is useful in order to assess the stability of the disaggregation results.
 
-# Bug fixes
+The ebrisk calculator accepts a new parameter called
+`minimum_asset_loss`: by specifying it, losses below the threshold are
+discarded in the computation of the aggregate loss curves. This does
+not improve the speed of the calculation much, but saves a substantial
+amount of memory. Notice that in the calculation of
+average losses and total losses the parameter
+`minimum_asset_loss` is ignored: losses are not discarded and the
+results you get are correct. It is only the aggregate loss curves that
+are approximated. The parameter is experimental and it is there for
+testing purposes; it may go away in the future.
+
+There is a new `event_based_damage` calculator which for the moment
+should be considered experimental. It is like a `scenario_damage`
+calculator working on a set of ruptures instead of a
+single rupture. It has the same features of a `scenario_damage`, including
+the ability to compute consequences given a consequence model.
+If you are interested in using it you should contact us.
+
+In order to support the `event_based_damage` calculator, the
+`scenario_damage` calculator has been changed and now uses a different
+algorithm if the field `number` in the exposure is an integer for all
+assets. In that case the damage state distribution is an array of
+integers and not of floats as it was in the past. The
+models in the global risk mosaic use fractional numbers for the `number` field,
+so they will keep using the old algorithm, but you will get a
+warning. The documentation is here XXX.
+
+Finally, there are a couple of new experimental features:
+
+- amplification of hazard curves
+- amplification of ground motion fields
+
+These features are not documented yet, because they are not ready, but 
+interested people are encouraged to contact us.
```

Comment:
"These features are not documented yet, because they are not ready, but interested people are encouraged to contact us." ->"These features are not documented yet, because they are not ready. We will add information in due course."

Reply:
Done